sandbox-backend/src/main/java/com/exadel/sandbox/team5/dao/OrderDAO.java

```diff
@@ -57,6 +57,16 @@ SELECT COUNT(o.id) FROM `order` o
             """)
     List<Pair> getAllOrdersForTags();
 
+    @Query(value = """
+            SELECT new com.exadel.sandbox.team5.util.Pair(c.name, COUNT(o.id))
+            FROM Order o
+                JOIN o.discount d
+                JOIN d.tags t
```

Comment:
`JOIN` -> `LEFT JOIN` for all not required fields, in this case for tags

Reply:
done